src/anemoi/training/config/training/default.yaml

```diff
@@ -33,10 +33,34 @@ swa:
 # use ZeroRedundancyOptimizer ; saves memory for larger models
 zero_optimizer: False
 
-# dynamic rescaling of the loss gradient
-# see https://arxiv.org/pdf/2306.06079.pdf, section 4.3.2
-# don't enable this by default until it's been tested and proven beneficial
-loss_gradient_scaling: False
+# loss functions
+loss_functions:
+
+  # dynamic rescaling of the loss gradient
+  # see https://arxiv.org/pdf/2306.06079.pdf, section 4.3.2
+  # don't enable this by default until it's been tested and proven beneficial
+  loss_gradient_scaling: False
+
+
+  # loss function for the model
+  loss:
+    # loss class to initialise, can be anything subclassing torch.nn.Module
+    _target_: anemoi.training.losses.mse.WeightedMSELoss
+    # what to include in the loss class initialisation
+    include_node_weights: True
+    include_feature_weights: True
+    # other kwargs
+    ignore_nans: False
+
+  # loss function for metric calculation
+  metrics:
```

Comment:
I think it would be great to make metrics a list, so users can register multiple evaluation metrics.

Reply:
Agreed, will address in next commit